crates/shared/src/order_quoting.rs

```diff
@@ -660,11 +660,25 @@ impl From<&OrderQuoteRequest> for PreOrderData {
 
 impl From<&OrderQuoteRequest> for QuoteParameters {
     fn from(request: &OrderQuoteRequest) -> Self {
+        let verification = if request.from == Default::default() {
+            None
+        } else {
```

Comment:
Does it make sense to keep special logic for the from address being zero? The code around this is confusing to me because we don't document that zero from address has special meaning. `OrderQuoteRequest` is created by a user of the post quote api, in this API the from field is mandatory and we do not document any special behavior with zero.